pkg/clusterapi/registry_mirror.go

```diff
@@ -47,7 +48,7 @@ func registryMirrorConfig(registryMirrorConfig *v1alpha1.RegistryMirrorConfigura
 
 	if registryMirrorConfig.CACertContent != "" {
 		files = append(files, bootstrapv1.File{
-			Path:    fmt.Sprintf("/etc/containerd/certs.d/%s/ca.crt", registryAddress),
+			Path:    fmt.Sprintf("/etc/containerd/certs.d/%s/ca.crt", registryAddressMappings[constants.DefaultRegistryMirrorKey]),
```

Comment:
I think the function in the api package that creates this mapping needs to return a specialized type with a function for retrieving the default. I mention this because currently, using the builtin map, you're requiring all consumers to embed the key knowledge of `constants.DefaultRegistryMirrorKey`. 

E.g.

```go
type OCINamespaceMap map[string]string

// define GetDefault func
```

Reply:
I was gonna comment something similar. Can we make that `GetRegistryMirrorAddressMappings` (or a different one) to return a struct that has this info more normalized?
I would expect the caller to not have to care about the map content except to just iterate over it to read the duples.